engine/src/main/scala/cromwell/engine/backend/JobDescriptor.scala

```diff
@@ -24,7 +24,7 @@ sealed trait JobDescriptor[K <: CallKey] {
   def locallyQualifiedInputs: CallInputs
 }
 
-final case class BackendCallJobDescriptor(workflowDescriptor: WorkflowDescriptor,
+case class BackendCallJobDescriptor(workflowDescriptor: WorkflowDescriptor,
```

Comment:
Oh yeah, I hit this before. It's annoying we have to remove safety from live code to facilitate tests but... that's life, I guess.